src/components/player/details.jsx

```diff
@@ -1,11 +1,63 @@
 import React from 'react';
+import YT_API_KEY from '../../../config/api_key';
 
 class Details extends React.Component {
 
+  constructor(props) {
+    super(props);
+
+    this.state = { details: {} };
+  }
+
+  componentDidMount() {
+    this._fetchDetails();
+  }
+
+  _fetchDetails() {
+    return fetch(`https://www.googleapis.com/youtube/v3/videos?part=snippet&id=${this.props.videoId}&key=${YT_API_KEY.publicDataKey}`)
```

Comment:
can we throw these fetches into the util file? 

we can just combine `search_result_util.js`, `trending_util.js`, and `player_util.js` into one file. maybe call it `youtube_video_util.js`?